autoscheduler/scraper/tests/api_tests.py

```diff
@@ -11,15 +11,15 @@ class APITests(APITestCase):
     def setUp(self):
         self.client = APIClient()
         self.courses = [
-            Course(id='123123', dept='CSCE', course_num='181',
+            Course(id='CSCE181-201931', dept='CSCE', course_num='181',
                    title='Introduction to Computing', term='201931', credit_hours=3),
-            Course(id='123124', dept='CSCE', course_num='315',
+            Course(id='CSCE315-201931', dept='CSCE', course_num='315',
                    title='Programming Studio', term='201931', credit_hours=3),
-            Course(id='123125', dept='COMM', course_num='203',
+            Course(id='COMM203-201831', dept='COMM', course_num='203',
                    title='Public Speaking', term='201831', credit_hours=3),
-            Course(id='123126', dept='COMM', course_num='203',
+            Course(id='COMM203-201931', dept='COMM', course_num='203',
                    title='Public Speaking', term='201931', credit_hours=3),
-            Course(id='123127', dept='LAW', course_num='7500S',
+            Course(id='LAW7500s-202031', dept='LAW', course_num='7500S',
```

Comment:
```suggestion
            Course(id='LAW7500S-202031', dept='LAW', course_num='7500S',
```
While changing this doesn't have any effect on the tests passing/failing, this is what the id should actually be